Mage/src/main/java/mage/abilities/SpellAbility.java

```diff
@@ -306,7 +306,7 @@ public Card getCharacteristics(Game game) {
     }
 
     public static SpellAbility getSpellAbilityFromEvent(GameEvent event, Game game) {
```

Comment:
it seems this method is used in a bunch of places for when comparing names as morphs have no name. Can you add some sort of explanatory comment on its usage?